detection-rules/link_new_domain_in_link_first_time_sender.yml

```
@@ -0,0 +1,22 @@
+name: "Link: New domain (<7d) in link from first-time sender"
```

```suggestion
name: "New link domain (<7d) from first-time sender"
```

i think we can simplify a bit